core/src/main/java/tachyon/Users.java

```diff
@@ -109,25 +78,12 @@ public String getUserUfsTempFolder(long userId) {
   }
 
   /**
-   * Get how much space quote does a user own.
-   *
-   * @param userId The queried user.
-   * @return Bytes the user owns.
-   */
-  public long ownBytes(long userId) {
-    synchronized (mUsers) {
-      UserInfo tUser = mUsers.get(userId);
-      return tUser == null ? 0 : tUser.getOwnBytes();
-    }
-  }
-
-  /**
    * Remove <code> userId </code> from user pool.
    *
    * @param userId The user to be removed.
    * @return The space quote the removed user occupied in bytes.
```

Comment:
Since this is public APIs, could you update the Javadoc that this method no longer return value?